resources/compatible-domains.json

```diff
@@ -64,5 +64,6 @@
     "payfit.com",
     "tiktok.com",
     "yahoo.co.jp",
-    "yahoo.com"
+    "yahoo.com",
+    "monespacesante.fr"
```

Comment:
can you order it please :)

Reply:
The list is not ordered at all

Reply:
Want me to order the entire list in the same PR?

Reply:
oh damn, i thought i ordered it at some point, yes please do then 👍

Reply:
done